generators/generator-adaptive-bot/generators/app/templates/assets/appsettings.json

```diff
@@ -1,7 +1,29 @@
 {
   "MicrosoftAppPassword": "",
   "MicrosoftAppId": "",
-  "applicationInsights": {
-    "InstrumentationKey": ""
+  "defaultLocale": "en-US",
+  "runtimeSettings": {
```

Comment:
@pavolum this section was added to all configs since this is the default minimum stuff. However if composer provides a 'base' setting, we could move this runtimeSettings node to composer and only keep 'additive' stuff in these appsettings.json. So it depends more on what composer does for us, how much it knows about our settings and what the design is on that regard.

In the long term, I'd love to get composer to know as little as possible about our config shape. The composer runtime plugin can know, but not he rest of composer ideally. Thoughts?

Reply:
Discussed with Patrick, it's fine as-is for the time being.

Reply:
Makes sense to me! At the moment Composer does not provide much of a base set of settings for the runtime obj which does lead to the template generators themselves declaring a similar set of boilerplate config in the runtime setting. 
As far as 'composer to know as little as possible about our config shape' that is the plan for the next release when we focus a bit more on non Composer environments like running F5 in VS.